website/src/PageWithSidebar.js

```diff
@@ -62,15 +62,14 @@ class PageWithSidebar extends Component {
                 state: route,
                 dispatch,
               });
-              const linkName = router.getScreenConfig(childNavigation, 'linkName');
-              const icon = router.getScreenConfig(childNavigation, 'icon');
+              const options = router.getScreenOptions(childNavigation, {});
               const isActive = state.index === i;
               return (
                 <span key={i}>
                   <LinkableLi
                     to={route.routeName}
-                    className={"pt-menu-header "+icon+' '+(isActive?'active':'')}>
-                    <h6>{linkName}</h6>
+                    className={"pt-menu-header "+options.icon+' '+(isActive?'active':'')}>
+                    <h6>{options.linkName}</h6>
```

Comment:
Here's a nice example of when a router will produce an option that is non-standard according to our other navigators. I wish the website had better flow typing so we could actually verify that the flow types work properly.